Fix and update README files. (#33)
ferPrieto/TimelineView: timelineview/README.md

TimelineView provides a synchronized scrolling experience with two content views that move in complementary directions. The component is built with Jetpack Compose for modern Android development and is perfect for timeline-based applications like audio/video editing, progress visualization, and data browsing.

## Installation

> ### 1. Add the library in the app-level `build.gradle` file 

Reviewer: ```suggestion
### 1. Add this dependency in `build.gradle` (app-level)
```

Kotlin: 

```kotlin
implementation("com.github.ferPrieto:timelineview:LATEST_VERSION_NUMBER")
```

Groovy:

```groovy
implementation 'com.github.ferPrieto:timelineview:LATEST_VERSION_NUMBER'
```

> If you're using a `libs.versions.toml` file:

Reviewer: ```suggestion
If you are using Gradle version catalog, through a `libs.versions.toml` file:
```

1. Add `timelineview = "LATEST_VERSION_NUMBER"` in the `[versions]` section.
2. Add `timelineview = { group = "com.github.ferPrieto", name = "timelineview", version.ref = "timelineview" }` in the `[libraries]` section
3. Add `implementation(libs.timelineview)` in the app-level `build.gradle` file

### 2. Declare Jitpack

> You need to declare Jitpack in your `settings.gradle` file because that's what's used to publish this library

Reviewer: ```suggestion
You must include `jitpack.io` in your `settings.gradle` file, since it's a public dependency```
